src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -1049,7 +1065,12 @@ public void getDocContent(Request req, Response resp)
     doCategories(documentManagement, node, resp);
     doNodeFeatures(node, resp);
     doNodeProperties(documentManagement, node, resp);
-    resp.setDisplayUrl(getDisplayUrl(node.getType(), node.getID()));
+    try {
+      resp.setDisplayUrl(getDisplayUrl(node.getType(), node.getID()));
+    } catch (URISyntaxException e) {
+      log.log(Level.WARNING, "Invalid Display URL for object ID "
```

Comment:
So you're basically sweeping these under the rug because they shouldn't ever happen? Maybe a comment to that effect, and since you're relying on this, more reason to include objectActions.keySet in the set of types to validate in init.